tensorflow/lite/micro/kernels/kernel_util.cc

```diff
@@ -13,13 +13,33 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-#include "tensorflow/lite/micro/kernels/kernel_util.h"
-
 #include "tensorflow/lite/c/common.h"
+#include "tensorflow/lite/micro/kernels/kernel_util.h"
```

Comment:
We'll probably want to revert the header order back to the original here, since kernel_util.h should be able to compile without common.h being included first, and including it first is a way to check that.

Reply:
clang-format3.9 produced this output ,anyway i will revert back